### DISCUSSION TITLE
CSS for image on website

### DISCUSSION THREAD

**Discussion by riishabbh** [atom/atom#22525] [Q&A]
What is the CSS for the image on atom.io? I like it and is it possible to use it?
Below is a screenshot taken from atom.io showing it.

<img width="732" alt="Screen Shot 2021-06-01 at 9 09 21 PM" src="https://user-images.githubusercontent.com/85147481/120422716-b569a680-c31d-11eb-8376-6f5fd0e79969.png">

Sorry if this is simple because I am new to GitHub and atom.

**Comment by ThatXliner** [comment]
Probably [transition](https://www.w3schools.com/css/css3_transitions.asp) + [`:hover`](https://www.w3schools.com/cssref/sel_hover.asp). You can always open the dev console (`cmd` + `option` + `i`) or right click > inspect element

**Reply by riishabbh** [reply]
ill check that out

**Reply by riishabbh** [reply]
Oh wait I am not asking for the thing that whenever I hover over the image it turns white, I am talking about the yellow CSS.

**Reply by ThatXliner** [reply]
[This](https://www.w3schools.com/cssref/css3_pr_filter.asp), maybe? Or [this?](https://css-tricks.com/almanac/properties/f/filter/)
